lib/ffmpeg/presets/aac.rb

```diff
@@ -66,13 +72,18 @@ def initialize(
         name: nil,
         filename: nil,
         metadata: nil,
+        threads: FFMPEG.threads,
         audio_bit_rate: '128k',
         &
       )
+        @threads = threads
         @audio_bit_rate = audio_bit_rate
         preset = self
 
         super(name:, filename:, metadata:) do
+          threads preset.threads if preset.threads
+          format_name 'mp4'
+          brand 'M4A '
```

Comment:
do we need the extra space at the end of the string?

Reply:
yup it's funny, but that's the official value 😄